migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,74 @@
+[id='migration-from-version-3-to-4-overview']
+= Migration from version 3 to 4 overview
+
+{product-title} 4 clusters are different from {product-title} 3 clusters. {product-title} 4 clusters contain new technologies and functionality that result in a cluster that is self-managing, flexible, and automated To know more about migrating from {product-title} 3 to 4 see,
```

Comment:
>and automated To know more about migrating from {product-title} 3 to 4 see,
does it miss a full stop before `To know`?

Reply:
The link was missing after `see,` fixed it.